dfm/description/matchcounter.py

```diff
@@ -0,0 +1,129 @@
+from collections import defaultdict
+
+from spacy.language import Language
+from spacy.matcher import Matcher
+from spacy.tokens import Doc
+
+from typing import Dict, Iterable, List, Optional
+
+
+class MatchCounter:
```

Comment:
The name of this file should be `match_counter`, or the class should be `Matchcounter`. I prefer the former for obvious reasons.

Reply:
Obv. reason here being consistentency with the casing (indicating that it is two words).